awscli/clidriver.py

```diff
@@ -379,15 +366,17 @@ def __call__(self, args, parsed_globals):
 
     def _create_command_table(self):
         command_table = OrderedDict()
-        service_object = self._get_service_object()
-        for operation_object in service_object.operations:
-            cli_name = xform_name(operation_object.name, '-')
+        service_model = self._get_service_model()
```

Comment:
Yeah that sounds good with me to remove. I believe I added it because there was a ``service_object`` property so I figured a parallel was going to be needed later down the road. The best option to avoid using ``service_model`` is to check whether the command is an instance of a ``ServiceCommand``.